### ISSUE TITLE
NodeValue.expression fails on DataFrame objects

### ISSUE THREAD

**Issue by Lehnchen** [alexmojaki/birdseye#92]
```
for i, (formatted_name, label) in enumerate(zip(val.columns.format(),
                                                            val.columns)):
```

produces an error if used with DataFrames: 
```

[...]bird.py", line 1045, in expression
    for i, (formatted_name, label) in enumerate(zip(val.columns.format(sparsify=False),
TypeError: format() got an unexpected keyword argument 'sparsify'
```

(birdeye=0.9.0, pandas: 1.2.4)

**Comment by alexmojaki** [comment]
Fixed in 0.9.1

**Comment by alexmojaki** [comment]
Thanks for the report!
